Src/Support/GoogleApis.Auth.PlatformServices_Shared/OAuth2/PromptCodeReceiver.cs

```diff
@@ -41,7 +40,13 @@ public string RedirectUri
             var authorizationUrl = url.Build().ToString();
 
             Logger.Debug("Open a browser with \"{0}\" URL", authorizationUrl);
```

Comment:
Move this line into the Process.Start part? We're not opening a browser in .NET Core, after all.